examples/tests/valid/expressions_string_interpolation.test.w

```diff
@@ -8,13 +10,35 @@ let reallyCoolString = "{number}{emptyString}\n{coolString}\n\{empty_string}{"st
 let beginingWithCoolStrings = "{regularString} {number} <- cool";
 let endingWithCoolStrings = "cool -> {regularString} {number}";
 
-assert("{1+1}" == "2");
-assert("\{1+1}" == "\{1+1}");
-assert("\{1+1}" != "2");
-assert("\{1+1}" != "\{2}");
+let nonInterpolated = #"a non { { {interpolated } } } strin{ g }";
+
+let nonInterpolatedJson = {a: nonInterpolated, b: #"this {one} too"};
+
+expect.equal("{1+1}", "2");
+expect.equal("\{1+1}", "\{1+1}");
+expect.notEqual("\{1+1}", "2");
+expect.notEqual("\{1+1}", "\{2}");
+log("\{1+1\} {number}");
+log(nonInterpolated);
+log("this is '{nonInterpolated}'");
```

Comment:
Are these logs on purpose? Maybe make them into an `expect`?

Reply:
yes, I added some expects, and two logs (since it's a common function that accepts a string and I wanted to see if we can pass the non-interpolated-string as an argument)